scripts/security-events.ts

```
@@ -0,0 +1,39 @@
+// security-events.ts
+import { PostHog } from 'posthog-node';
+
+const posthogApiKey = process.env.POSTHOG_API_KEY;
+const posthog = posthogApiKey ? new PostHog(posthogApiKey) : null;
+
+export const logSecurityScan = async (
+  scanType: string,
+  results: {
+    vulnerabilities?: any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)